src/api/sort.js

```diff
@@ -0,0 +1,12 @@
+export const  sortByAge = (ObjA, ObjB) =>  ObjA.age - ObjB.age;
+
+export const sortByName = (ObjA, ObjB) =>  {
+    const a = ObjA.name.toLowerCase(),
+      b = ObjB.name.toLowerCase();
+      
+    if(a > b)
+      return 1;
+    if(a < b)
+      return -1;
+  return 0
```

Comment:
```suggestion
  return ObjA.name.localeCompare(ObjB.name);
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

Reply:
nice method =)
done